docs/source/get_started/kedro_concepts.md

```diff
@@ -63,20 +63,53 @@ The Kedro Data Catalog is the registry of all data sources that the project can
+├── requirements.txt # Project dependencies file
+```
+
+### Example Kedro project structure (no tools selected)
+
+If you select no tools, the resulting structure will be simpler:
 
 ```
-project-dir         # Parent directory of the template
-├── .gitignore      # Hidden file that prevents staging of unnecessary files to `git`
-├── conf            # Project configuration files
-├── data            # Local project data (not committed to version control)
-├── docs            # Project documentation
-├── notebooks       # Project-related Jupyter notebooks (can be used for experimental code before moving the code to src)
-├── pyproject.toml  # Identifies the project root and contains configuration information
-├── README.md       # Project README
-└── src             # Project source code
+project-dir          # Parent directory of the template
+├── conf             # Project configuration files
+├── notebooks        # Project-related Jupyter notebooks (can be used for experimental code before moving the code to src)
+└── src              # Project source code
+├── .gitignore       # Hidden file that prevents staging of unnecessary files to `git`
+├── pyproject.toml   # Identifies the project root and contains configuration information
+├── README.md        # Project README
+├── requirements.txt # Project dependencies file
 ```
 
+### Tool selection and resulting structure
+
+During `kedro new`, you can select which tools to include in your project. Each tool adds specific files or folders to the project structure:
+
+- **Lint (Ruff)**: Modifies the `pyproject.toml` file to include Ruff configuration settings for linting. It sets up `ruff` under `[tool.ruff]`, defines options like line length, selected rules, and ignored rules, and includes `ruff` as an optional `dev` dependency.
+- **Test (Pytest)**: Adds a `tests` folder for storing unit and integration tests, helping to maintain code quality and ensuring that changes in the codebase do not introduce bugs. For more information about testing in Kedro, visit the [Automated Testing Guide](https://docs.kedro.org/en/stable/development/automated_testing.html).
+- **Log**: Allows specific logging configurations by including a `logging.yml` file inside the `conf` folder. For more information about logging customisation in Kedro, visit the [Logging Customisation Guide](https://docs.kedro.org/en/stable/logging/index.html).
+- **Docs (Sphinx)**: Adds a `docs` folder with a Sphinx documentation setup. This folder is typically used to generate technical documentation for the project.
+- **Data Folder**: Adds a `data` folder structure for managing project data. The `data` folder contains multiple subfolders to store project data. We recommend you put raw data into `raw` and move processed data to other subfolders according to the [commonly accepted data engineering convention](https://towardsdatascience.com/the-importance-of-layered-thinking-in-data-engineering-a09f685edc71).
```

Comment:
Can we rephrase this so it's more clear that this is _Kedro's_ data engineering convention, and not some universal standard?